portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,80 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.Entity;
+
+public class Comment {
+    private final String text;
+    private final String user;
+    private final long timestamp;
+    private int upvotes;
+    private int downvotes;
+    private long key;
```

Comment:
Could these fields also be `final`?

Reply:
done

Reply:
actually, the values can change when a user up/downvotes a comment...